### PR TITLE
Improved Index Heads

### PR DESCRIPTION
**Pull request by MilesBHuff**
Just some improvements to the `<head/>` portion of the two `index*.html` files.

### CODE REVIEW

File: index-electron.html [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
-        window.dataLayer = window.dataLayer || [];
-        function gtag() {
-            dataLayer.push(arguments);
-        }
-        gtag('js', new Date());
-        gtag('config', 'UA-66229740-5');
-    </script>
+        <!-- MISC -->
+        <link  rel="manifest" type="application/manifest+json" href="/static/manifest.webmanifest" />
+		<meta name="background-color" content="#000" />
```
Comment (jjppof) [inline index-electron.html:19]: Need to fix indentation here 😁

File: index.html [hunk 2]
```diff
@@ -1,235 +1,229 @@
...
-    <script>
-        //Prevent arrows to scroll the page
-        window.addEventListener("keydown", function(e) {
-            if ([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) { // space and arrow keys
-                e.preventDefault();
-            }
-        }, false);
+        <!-- MISC -->
+        <link  rel="manifest" type="application/manifest+json" href="/static/manifest.webmanifest" />
+		<meta name="background-color" content="#000" />
```
Comment (jjppof) [inline index.html:19]: Also here!

File: index-electron.html (hunk 1 above)
Comment (MilesBHuff) [reply index-electron.html:19]: lool whoops

File: index.html (hunk 2 above)
Comment (MilesBHuff) [reply index.html:19]: Too bad Prettier doesn't do html files!

**Review by jjppof — COMMENTED** [review]
Just indentations stuff :)

File: index.html (hunk 2 above)
Comment (jjppof) [reply index.html:19]: Yeah :/

**Comment by MilesBHuff** [comment]
Fixed.

**Review by jjppof — APPROVED** [review]

**Pull request merged**